configs/yaml_serializer.go

```diff
@@ -113,10 +109,10 @@ func (config *Configuration) SerializeToYAML() ([]byte, error) {
 			Password: config.ProxyPassword,
 		}
 	}
-	if len(config.BoardManagerAdditionalUrls) > 1 {
```

Comment:
I've written `1` because when a new configuration is created with `configs.NewConfiguration` the `BoardManagerAdditionalUrls` is filled with the default `package_index.json` url.
I don't know if it's better to output the default index or not, maybe we should treat it as a special case and leave only 3rd party urls in `BoardManagerAdditionalUrls`?

Reply:
I mean it's called AdditionalUrls, so it makes sense to avoid it?

Reply:
I agree, but then we should check when it's used and add an extra call for the default URL.

Reply:
Sounds like something for another PR, I just restored the previous behaviour